src/main/java/com/github/fge/jsonpatch/JsonPatch.java

```diff
@@ -123,11 +123,9 @@ public JsonPatch(final List<JsonPatchOperation> operations)
      * @throws NullPointerException input is null
      */
     public static JsonPatch fromJson(final JsonNode node)
-        throws IOException
-    {
+            throws IOException, JsonPatchException {
```

Comment:
this smells, usually you have one exception type coming out and either wrap the IOException or don't return a more specifically typed one.